sunscreen/src/compiler.rs

```diff
@@ -8,7 +8,7 @@ use std::collections::{HashMap, HashSet};
 use std::marker::PhantomData;
 use sunscreen_fhe_program::FheProgramTrait;
 use sunscreen_runtime::{marker, CompiledFheProgram, Fhe, FheZkp, Zkp};
-use sunscreen_zkp_backend::CompiledZkpProgram;
+use sunscreen_zkp_backend::{BackendField, CompiledZkpProgram, ZkpBackend};
```

Comment:
TODO: Prevent user from calling compile until they have specified either a ZKP or FHE program.

Doing so today returns `Error::NoPrograms` at runtime, but we can make this a compile-time error.

Reply:
Another time.